src/main/resources/db/changelog/changes/4-create-info-type-table-changelog.sql

```diff
@@ -0,0 +1,17 @@
+--liquibase formatted sql
+
+--changeset noyan:1
+
+CREATE TABLE `type` (
```

Comment:
please change the name of the table to "types" to be more consistent.

Reply:
I changed the name of table as "types"